libs/core/src/mutex.cpp

```diff
@@ -17,144 +17,129 @@
 //------------------------------------------------------------------------------
 
 #include "core/mutex.hpp"
+#include "moment/clocks.hpp"
 
+#include <cassert>
 #include <iostream>
 #include <sstream>
 
-namespace fetch {
+using namespace std::chrono_literals;
 
-#ifdef FETCH_DEBUG_MUTEX
```

Comment:
@ejfitzgerald do we want to turn on the debug mutex by default?

Reply:
No I don't think so. Lets leave it as it currently is as an option to be enabled